spec/integration/solo/solo_spec.rb

```diff
@@ -163,7 +163,7 @@
         ruby_block "sleeping" do
           block do
             retries = 200
-            while IO.read(Chef::Config[:log_location]) !~ /Chef client .* is running, will wait for it to finish and then run./
+            while IO.read(Chef::Config[:log_location]) !~ /#{Chef::Dist::PRODUCT} client .* is running, will wait for it to finish and then run./
```

Comment:
I don't think it's going to be helpful to include distro-variances in these unit tests. They care only about the project, not the distribution system. I would use `.*` in places to make the tests work under any configuration.

Reply:
just pushed some fixes for this too